Client.Core.Test/Client.Core.Test.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
     <PropertyGroup>
-        <TargetFrameworks>netcoreapp3.1;netcoreapp5.0;netcoreapp6.0</TargetFrameworks>
+        <TargetFrameworks>netcoreapp3.1;net5.0;net6.0</TargetFrameworks>
```

Comment:
net5.0 is not supported and netcoreapp3.1's last date is next tuesday: https://dotnet.microsoft.com/en-us/platform/support/policy/dotnet-core

maybe net6.0 and net7.0 support is enough

Reply:
We don't want to remove support for old versions yet.

Reply:
ok. i think net7.0 could be supported at the same time now that it's been released.

Reply:
You are right... I've updated the sources